apis/v1beta1/.import-restrictions

```diff
@@ -0,0 +1,5 @@
+rules:
+  - selectorRegexp: sigs[.]k8s[.]io/controller-runtime
```

Comment:
I tried to break it by adding an import but I couldn't get it to fail. Does it work for you?

Reply:
I tried to revert the changes to remove the changes regarding `SchemaBuilder` in an earlier PR and it breaks as expected.
You may add
`	SchemeBuilder = &scheme.Builder{GroupVersion: GroupVersion} ` to `apis/v1beta1/groupversion_info.go`.

It should give you an output like
```
Enforcing imports in source codes under the following directories: ./apis/v1alpha3,./apis/v1alpha4,./apis/v1beta1,./apis/vmware/v1beta1
E0908 18:12:34.342983   56319 main.go:41] Error: Failed executing generator: some packages had errors:
errors in package "./apis/v1beta1":
import sigs.k8s.io/controller-runtime/pkg/scheme has forbidden prefix sigs.k8s.io/controller-runtime
the following imports did not match any allowed prefix:
  sigs.k8s.io/controller-runtime/pkg/scheme
  sigs.k8s.io/controller-runtime/pkg/scheme
```

Reply:
I'll try again. I tried exactly this, probably did something wrong

Reply:
@laozc Me and @chrischdi can neither produce a failure locally nor via Prow. 

This PR should fail, right? https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/2344/files#diff-be33a5a021a191e201be9e68902a2bb0181f88d46fd9f61d6f0c116888239172